### DISCUSSION TITLE
Welcome to DiffusionToolkit Discussions!

### DISCUSSION THREAD

**Discussion by RupertAvery** [RupertAvery/DiffusionToolkit#189] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rpillala** [comment]
I'm an end user and I just use it to view and find my sd generations. This is a very useful program thanks for working on it!
